### DISCUSSION TITLE
Add optional String validateSettings(Map<String, String> propertiesMap) method too ActionHook

### DISCUSSION THREAD

**Discussion by kschmiechen** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.interceptor-manager#88] [Ideas]
To avoid invalid settings, the interceptor manager could call an optional method `String validateSettings(Map<String, String> propertiesMap)` of the hook each time its settings are updated in the administration panel. The method returns `null` if the settings are valid and a `message` if the settings are invalid.

**Comment by Jumas** [comment]
Hello Kevin, thanks for the idea.
It is implemented with the small change in the method signature.
